armi/reactor/tests/test_parameters.py

```diff
@@ -68,16 +68,15 @@ class Mock(parameters.ParameterCollection):
 
     def test_writeSomeParamsToDB(self):
         """
-        This test tests the ability to specify which parameters should be
+        This tests the ability to specify which parameters should be
         written to the database. It assumes that the list returned by
         ParameterDefinitionCollection.toWriteToDB() is used to filter for which
         parameters to include in the database.
 
         .. test:: Test to restrict some parameters from being written to the database.
```

Comment:
Let's make this even shorter. Something like:

> .. test:: Restrict parameters from DB write

One, I don't think we want to start all of these with "Test to".  But, also, I think if these are really short stubs, people will be less likely to leak priority information in the future.